lib/clone.js

```diff
@@ -49,21 +52,24 @@ const cloneTask = (sshGitHubUrl, repoName, progressBar) => {
 const clone = (services, task = cloneTask) => {
   process.stdout.write(`Cloning repos...${newline}`)
 
-  const gitHubDetails = getGitHubDetails(services)
+  const gitHubRepoInfo = getGitHubDetails(services)
   const progressBar = new Progress(':percent [:bar] (:current/:total)', {
-    total: gitHubDetails.length,
+    total: gitHubRepoInfo.length,
     clear: true
   })
+  const githubInfoParentArray = chunkArray(gitHubRepoInfo, numberOfCPUs * 4)
```

Comment:
I have just pulled the latest on this and it works fine on my Mac. My Mac did go through a crazy period which resulted in this PR but it seems to have stabilised so maybe it was a bit of an edge case but worth doing for scaling.